### ISSUE TITLE
Validate error

### ISSUE THREAD

**Issue by andim27** [nylo-core/nylo#55]
nylo_framework: ^5.0.2
Example: 
```
  validate(
        rules: {
          "name": "not_empty",
          "email": "email",
          "phone number": "regex:(^[+\(\d+\)]+)"
        },
        data: {
          "name": nameFieldValue,
          "email": emailFieldValue,
          "phone number": phoneFieldValue,
        },
        messages: {
          "Name": "Name failed",
          "Email address": "Email failed",
          "Phone number": "Correct phone number format: +(**)**********"
        },
        onSuccess: () {
        },
        onFailure: (Exception exception) {
          print('Validate error: ${exception.toString()}');
        },
        showAlert: true, 
        alertStyle: ToastNotificationStyleType.DANGER);
  }
```
Alert doesn't show correct messages.It works only for first field

**Comment by agordn52** [comment]
Working on a fix for this @andim27, thanks for highlighting it

**Comment by agordn52** [comment]
Hi @andim27,

This is fixed in **v5.1.0** 🚀
